manager/attestation_policy.go

```diff
@@ -25,7 +25,12 @@ import (
 const defGuestFeatures = 0x1
 
 func (ms *managerService) FetchAttestationPolicy(_ context.Context, computationId string) ([]byte, error) {
```

Comment:
can we write the output of the attestation policy bin to std out rather than json. that  way manager can read and parse the command output and not worry about location of the output json fille